csharp/src/Apache.Arrow.Flight.Sql/Apache.Arrow.Flight.Sql.csproj

```diff
@@ -5,7 +5,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Grpc.Tools" Version="2.67.0" PrivateAssets="All" />
+    <PackageReference Include="Grpc.Tools" Version="2.66.0" PrivateAssets="All" />
```

Comment:
This version number shouldn't be reverted

Reply:
Understood. Starting fixups.